server/src/com/cloud/network/vpn/RemoteAccessVpnManagerImpl.java

```diff
@@ -613,6 +614,10 @@ public void doInTransactionWithoutResult(TransactionStatus status) {
             sc.setParameters("username", username);
         }
 
+        if (keyword!= null) {
+            sc.setParameters("username",  "%" + keyword + "%");
+        }
+
         Pair<List<VpnUserVO>, Integer> result = _vpnUsersDao.searchAndCount(sc, searchFilter);
```

Comment:
@syed : If both are set, we are giving preference to keyword as the search will happen using keyword query param. This is in according with other existing APIs. Keyword is the request parameter for : List by keyword functionality.